ansible/eda-server
# [AAP-11111]: Rulebooks filter by project id

### Jira Ticket: [AAP-11111](https://issues.redhat.com/browse/AAP-11111) ###

### Purpose: ###
- Add filtering by project id functionality to list rulebooks endpoint
- filter is `project`

### Testing Instructions: ###
1. populate the DB with a couple of projects that contain rulebooks
2. Hit the list rulebooks page to ensure they are populated: [GET-rulebooks](http://localhost:8000/api/eda/v1/rulebooks/)
3. filter by project: `/api/eda/v1/rulebooks/?project_id={id}`
   - results should only return projects with that id
4. test with none existent id: `/api/eda/v1/rulebooks/?project_id={id-does-not-exist}`
   - results should return an empty array []
5. Ensure the [docs](http://127.0.0.1:8000/api/eda/v1/docs) include the project filter parameter
